types/set/fundamental/read.C

```diff
@@ -0,0 +1,68 @@
+#include <ROOT/REntry.hxx>
+#include <ROOT/RNTupleReader.hxx>
+
+using ROOT::Experimental::REntry;
+using ROOT::Experimental::RNTupleReader;
+
+#include <cstdint>
+#include <fstream>
+#include <ostream>
+#include <set>
+#include <string>
+#include <string_view>
+
+using Set = std::set<std::int32_t>;
+
+static void PrintSetValue(const REntry &entry, std::string_view name,
+                          std::ostream &os, bool last = false) {
+  Set &value = *entry.GetPtr<Set>(name);
+  os << "    \"" << name << "\": [";
+  bool first = true;
+  for (auto element : value) {
```

Comment:
Actually, is the iteration order defined here? Do we want an explicit construction of a `std::vector` (?) and sort it? If not, the output file might change from execution to execution...

Reply:
Ah, I commented on the wrong test: the question is mostly relevant for `std::unordered_[multi]set`, but for symmetry we may want to do it for all tests.